changelogs/2022-02-changelog.md

```diff
@@ -0,0 +1,32 @@
+# EMAP Release
+
+**Date: 2022-02-07 Changes in this release**
+
+---
+
+
+### Changes/fixes
+
+* Temporal columns are now all timezone aware, this has been enforced with automated testing of any timezone field
+* Hl7 feed only parses new patient infections
+* Patient infections that are associated with an encounter have a hospital_visit_id
+* Improved speed of load times by updating caching
+* Fixes Log4j exploits
+* Coded flowsheets are now parsed as the raw id values given in the Hl7 feed
+
```

Comment:
oh and we need changes to visit_obvervation now having a source_system column. Then have changes to visit_observation_type columns. Processing of flowsheets now only has one visit_observation_type for both the hl7 and caboodle types.

Reply:
Should we say here that we therefore maintain two identifiers? it used to be just one ...

Reply:
Oh yeah that wasn't a suggestion of wording to put in, just an overview of what was missing!